lib/context.js

```diff
@@ -15,11 +15,17 @@ function isExpired() {
   return tokenSet.call(this).expired();
 }
 
-async function refresh() {
+async function refresh(params) {
```

Comment:
`refresh({ tokenEndpointParams })` Let's make this a config object and also call it `tokenEndpointParams`